test/ci/try_assert.cc

```diff
@@ -0,0 +1,8 @@
+#include <cassert>
+#include <iostream>
+
+int main(int, char**) {
+  assert(false);
+
+  std::cout << "Assert did not exit!" << std::endl;
+}
```

Comment:
Thanks, I'll add an explicit `return 0` for completeness. When `TILEDB_ASSERTIONS=ON` this will exit due to the assert with a OS (windows, POSIX)-specific error code.